### PR TITLE
Text utilities and lists

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (61)
Screenshot (62)
Screenshot (63)
Screenshot (64)
Screenshot (65)
Screenshot (66)
Screenshot (67)
Screenshot (68)
Screenshot (69)

This is my text utility and list component. I still have to add carbon code.

### CODE REVIEW

File: Components/Text utilities/text-utility.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    font-size: x-small;
+}
+
+.xxsmall-text {
+    font-size: xx-small;
+}
+
+.w-700 {
+    min-width: 31.25rem;
+    margin: 0.5rem 0rem;
```
Comment (zoyatanweer) [inline Components/Text utilities/text-utility.css:42]: use px here

File: Components/Text utilities/text-utility.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    padding: 0rem;
+}
+
+.gray-text {
+    color: gray;
```
Comment (zoyatanweer) [inline Components/Text utilities/text-utility.css:47]: use variable here for color

File: css/doc.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+.title-content {
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 2rem;
+    margin-right: 12.5rem;
+}
+
+.w-800 {
+    width: 50rem;
+    margin: 0.625rem 0.625rem 2.5rem 0.625rem;
```
Comment (zoyatanweer) [inline css/doc.css:66]: use px herre

**Review by zoyatanweer — APPROVED** [review]

**Review by itsmebipul00 — APPROVED** [review]
Nice Work

**Comment by itsmebipul00** [comment]
Branch has conflicts though!!!!

**Pull request merged**